packages/ui/src/components/SelectInputV2/SelectBar.tsx

```diff
@@ -54,7 +58,7 @@ const Placeholder = styled(Text)`
 user-select: none;
 `
 
-export const StyledInputWrapper = styled(Stack)<{
+const StyledInputWrapper = styled(Stack)<{
```

Comment:
There are some visual glitches if I play with the multi select:

https://github.com/user-attachments/assets/d05a4b18-4088-44e5-9ff1-7cc87ee8a144